client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,44 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: #ffffff;
+  color: #ffffff;
+  border: 2px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  padding: none;
+  border:  none;
+  border-collapse: collapse;
+  display: block;
+  overflow-x: scroll;
+  overflow-y: scroll;
```

Comment:
Why not `overflow: auto` so that you only get each scrollbar if needed?

Reply:
Fixed, thanks.